_security-plugin/configuration/yaml.md

```diff
@@ -124,6 +124,12 @@ plugins.security.restapi.password_validation_regex: '(?=.*[A-Z])(?=.*[^a-zA-Z\d]
 plugins.security.restapi.password_validation_error_message: "Password must be minimum 8 characters long and must contain at least one uppercase letter, one lowercase letter, one digit, and one special character."
 ```
 
+The opensearch.yml file also contains the `plugins.security.allow_default_init_securityindex` property. When set to `true`, the security plugin uses default security settings if an attempt to create the security index fails when OpenSearch launches. Default security settings are stored in YAML files contained in the config folder. By default, this setting is `false`.
```

Comment:
I would clarify which "config" folder you mean. If it's `opensearch/config` then it might be good to call it out as a relative path like that. Or if the reader already knows from somewhere earlier in the document that "the config folder" is located there, then maybe add ticks so it's monospaced like `config`.

Reply:
Also, need to change it to `directory` instead of `folder`.  We don't have folders in Linux.

Reply:
Incorporated those suggestions. Good improvements.